include/rtc/rtp.hpp

```diff
@@ -0,0 +1,523 @@
+/**
+ * Copyright (c) 2020 Staz Modrzynski
+ * Copyright (c) 2020 Paul-Louis Ageneau
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#ifndef WEBRTC_SERVER_RTP_HPP
+#define WEBRTC_SERVER_RTP_HPP
+
+#include <cmath>
+#include <netinet/in.h>
```

Comment:
This include is not available on Windows, where you should instead include `winsock2.h` like in `rtcp.cpp`. Also, I think what you need here is actually `#include <arpa/inet.h>` for the `hton*` functions (the fact that `netinet/in.h` seems to include `arpa/inet.h` itself is more of a side-effect):
```
#ifdef _WIN32
#include <winsock2.h>
#else
#include <arpa/inet.h>
#endif
```